src/storage/file_manager.c

```diff
@@ -7961,7 +7960,6 @@ file_temp_alloc (THREAD_ENTRY * thread_p, PAGE_PTR page_fhead, FILE_ALLOC_TYPE a
 	pgbuf_fix (thread_p, &fhead->vpid_last_temp_alloc, OLD_PAGE, PGBUF_LATCH_WRITE, PGBUF_UNCONDITIONAL_LATCH);
       if (page_ftab == NULL)
 	{
-	  assert_release (false);
```

Comment:
Please notice that `file_temp_alloc` and `file_temp_reset_user_pages` disable interrupt.
Since we don't expect any error here, I'd like to propose to keep the assertions. 
Did you suffer from other errors?